config/environments/test.rb

```diff
@@ -1,10 +1,11 @@
+# The test environment is used exclusively to run your application's
+# test suite. You never need to work with it otherwise. Remember that
+# your test database is "scratch space" for the test suite and is wiped
+# and recreated between test runs. Don't rely on the data there!
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = true
```

Comment:
Can you also update this to `config.cache_classes = false` and add `config.action_view.cache_template_loading = true ` according to [this][1]

[1]: http://railsdiff.org/5.2.4.3/6.0.3.2

Reply:
Changing this seems to break the specs with https://github.com/rails/rails/issues/36363. 
In other PR's it seems we have kept it the same like this

Reply:
As discussed in Slack, this app doesn't seem to have Spring installed so we should leave it as true.